### PR TITLE
Events page

### PR DESCRIPTION
**Pull request by kianadk**
events
eventshover
eventsentercontest

### CODE REVIEW

File: react/frontpage/EventsContent.jsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
+// EventsContent.jsx
+var React = require('react');
+
+var Modal = require('react-bootstrap').Modal;
+var Input = require('react-bootstrap').Input;
+var FormControls = require('react-bootstrap').FormControls;
+var Button = require('react-bootstrap').Button;
+if (typeof Button === 'undefined' || Button === null) {
```
Comment (hsuregan) [inline react/frontpage/EventsContent.jsx]: What's this for? Curious

File: react/frontpage/EventsContent.jsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+var getBandName = function(desc){
+	return desc.substring(0, desc.indexOf("@") - 1);
+};
+
+var getVenue = function(desc){
+	return desc.substring(desc.indexOf("@") + 2);
+};
+
+var formatDate = function(dateString) {
```
Comment (hsuregan) [inline react/frontpage/EventsContent.jsx]: We should make functions like these accessible everywhere

File: react/frontpage/EventsContent.jsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+							</div>
+						</div>
+					);
+				})
+			}
+			</div>
+		);
+	}
+});
+
```
Comment (hsuregan) [inline react/frontpage/EventsContent.jsx]: I should start doing this too-- could you write a comment for what each react class does right before defining them?

File: react/frontpage/EventsContent.jsx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+	render: function(){
+		var hoverStyle = {display: "none"};
+		if(this.state.hover){
+			hoverStyle = {height: 200, width: 200, backgroundColor: "grey", opacity:"0.8", position: "absolute", bottom: 0, left: 0, right: 0, display: "inline"};
+		}
+		return(
+		<div>
+			<div className="event" onClick={this.showModal} onMouseEnter={this.mouseEnter} onMouseLeave={this.mouseLeave}>
+				<p className="bandName">{getBandName(this.props.event['summary'])}</p>
+				<img src="https://avatars3.githubusercontent.com/u/7256558?v=3&s=466" className="bandPic"/>
```
Comment (hsuregan) [inline react/frontpage/EventsContent.jsx]: are there any plans to use different images for different events?

File: react/frontpage/EventsContent.jsx (hunk 1 above)
Comment (kianadk) [inline react/frontpage/EventsContent.jsx]: My bad--that's just some debugging code that I forgot to get rid of

File: react/frontpage/EventsContent.jsx (hunk 3 above)
Comment (kianadk) [inline react/frontpage/EventsContent.jsx]: Yes, I'll start doing that

File: react/frontpage/EventsContent.jsx (hunk 4 above)
Comment (kianadk) [inline react/frontpage/EventsContent.jsx]: Yes, according to the mockups. I think we're confirming that with Promotions soon too.

**Comment by ChrisLaganiere** [comment]
Sending these pics to bryan for review.. thanks @kianadk 😁

**Comment by ChrisLaganiere** [comment]
> Love it! In terms of layout, can it look like a calendar? I hope to put other non giveaway content on there cuz people want their event to be promoted. If possible is there a way to differentiate between giveaway posts and like general events being promoted? My idea would be that you'd click the box and it'd redirect you to the event webpage

Here's feedback from Bryan, sounds like we want to have links go to a url for each one instead of an entering action. I like the layout, but maybe we can identify a little better what the dates are? We could have the date covering the photo and disappear to show the image when they hover, for example. We always want to try to make it mobile-friendly too, though 😛

**Comment by ChrisLaganiere** [comment]
merged into #50

**Pull request closed without merging**